com.microsoft.mrtk.accessibility/Utilities/ScreenReaderSettings.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Accessibility
+{
+    /// <summary>
+    /// 
```

Comment:
MIssing some summaries

Reply:
this is a too early file :smile: removing for now